homeassistant/components/fritzbox/sensor.py

```diff
@@ -107,7 +107,7 @@ class FritzSensorEntityDescription(
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
-        native_value=lambda device: device.power / device.voltage / 1000
+        native_value=lambda device: round(device.power / device.voltage / 1000, 4)
```

Comment:
Actually, this is a "virtual" entity, as it is not a native value provided by the device or service (but calculated in HA). That is not allowed. This sensor should be removed.

Reply:
It would be very helpful to see current in ampere out of the box from this device to improve user experience, because most other devices provide the same.
I don't see any convenient way to calculate current in HA. Is there an easy way by GUI to create a current entity by doing calculation including rounding? A rule would run into errors, if the device is renamed.
So why not provide calculated current out of the box?

Reply:
this discussion should be continued in https://github.com/home-assistant/core/pull/78324#discussion_r968832283